content/en/security_platform/_index.md

```diff
@@ -6,34 +6,13 @@ aliases:
   - security_monitoring
 ---
 
-<div class="alert alert-warning">
-Compliance Monitoring is currently available for private beta. Use this <a href="https://docs.google.com/forms/d/e/1FAIpQLScTA913tNGptcAeNNdWlvgECjekYoDVJwR-OkMtZYnJnq-FWw/viewform">form</a> to request access.
-</div>
-
 ## Overview
 
-{{< img src="security_monitoring/overview_top.png" alt="Datadog Security Monitoring" >}}
-
-Datadog Security and Compliance Monitoring unify developer, operation, and security teams through one platform. Use a single dashboard to display DevOps content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.
-
-{{< img src="security_monitoring/takeover_ex.png" alt="Slack Example"  style="width:75%;">}}
-
-Threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. You can clone any of the provided detection rules to change the configuration. You can also add a [new rule][3] from scratch to fit your specific use case.
-
-{{< img src="security_monitoring/explorer.png" alt="Security Signals Explorer"  >}}
-
-Enable runtime security and continuous Compliance Monitoring for your containers and hosts. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories on hosts or containers in real-time, or **Continuous Compliance** to check against your containers and Kubernetes clusters to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes.
-
-{{< img src="compliance_monitoring/compliance-dashboard.png" alt="Datadog Compliance Monitoring" >}}
+The Datadog Security Platform unifies developer, operation, and security teams through one platform. Use a single dashboard to display DevOps content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.
```

Comment:
Sentence is a bit long/confusing. Let's use this as a replacement for this paragraph:

Bring speed and scale to your production security operations. Datadog Cloud Security Platform delivers real-time threat-detection and continuous configuration audits across applications, hosts, containers, and cloud infrastructure. Coupled with the observability platform, Datadog brings unprecedented integration between security and operations aligned to the shared goals of the organization.